src/main/bash/commands/kobman-install.sh

```diff
@@ -52,25 +52,25 @@ function __kobman_create_environment_directory
 		local version_id=$2
 
 		cd "${KOBMAN_DIR}/envs"
-                mkdir -p kob_env_"${environment_name}"
-                cd kob_env_"${environment_name}"
-                touch current
+		mkdir -p kob_env_"${environment_name}"
+		cd kob_env_"${environment_name}"
+		touch current
 		destdir="${KOBMAN_DIR}/envs/kob_env_${environment_name}/current"
```

Comment:
@asa1997  destdir seems to be a file not a dir

Reply:
@EtricKombat please validate and confirm back

Reply:
@asa1997 @panickervinod : destdir is used to point to the current file , which is being re-used . for assigning the version values

Reply:
@asa1997 : can you please list out the uninstall commands to try out. So that i can test it out and review